package-builders/hypre.sh

```diff
@@ -73,19 +73,35 @@ function hypre_build()
       my_cflags="-g"
    fi
    echo "Building $pkg, sending output to ${pkg_bld_dir}_build.log ..." && {
+   local CUDA_MAKE_OPTS=()
+   if [[ -n "$CUDA_ENABLED" ]]; then
+      CUDA_MAKE_OPTS=(
+         "--with-cuda"
+         "--with-gpu-arch=70"
+         "--enable-device-memory-pool"
```

Comment:
I assume so but do we know if this is as performant as using Umpire? Also, does this disable the use of unified memory?

Reply:
No. We suggest using Umpire if you can. We probably will remove our device memory pool implementation in near future since even CUDA now has the "official" memory pool.

Reply:
The default for hypre is no unified memory (ref. docs). 
--enable-unified-memory Use unified memory for allocating the memory
                        (default is NO).
                        
If I recall correctly I think Umpire does improve the performance a bit, not sure by how what factor compared to the internal memory pool framework.

Reply:
@liruipeng do you know how much faster it is to use umpire?

Reply:
> @liruipeng do you know how much faster it is to use umpire?

Hard to give a number. It highly depends on the application.